Automattic/woocommerce-services
# Chore: phpcbf and phpcs will now check against WP and WC standards

## Description
Running `composer phpcbf <file>` doesn't fix the file. Same for `phpcs`. This PR fixes the `lint-staged` script as well as the `phpcbf` and `phpcs` composer scripts. These will now check against the following standards `--standard=WooCommerce-Core,WordPress-Core,WordPress-Extra`.

### Related issue(s)
N/A

### Steps to test
1. Run `composer phpcbf woocommerce-services.php`
2. Confirm that you see it auto fixed 4 things and 21 remaining items
```

PHPCBF RESULT SUMMARY
------------------------------------------------------------------------------------------------------------------------------------
FILE                                                                             FIXED  REMAINING
------------------------------------------------------------------------------------------------------------------------------------
..wordpress/wp-content/plugins/woocommerce-services/woocommerce-services.php     4      21
------------------------------------------------------------------------------------------------------------------------------------
A TOTAL OF 4 ERRORS WERE FIXED IN 1 FILE
------------------------------------------------------------------------------------------------------------------------------------
```

### Checklist
